src/index.js

```diff
@@ -0,0 +1,147 @@
+import _, { add, times } from 'lodash';
+import { clearAll } from './script.js';
+import taskCompleted from './script.js';
+import './style.css';
+
+const myContainer = document.getElementById('to-do-list');
+tasks.forEach((item) => {
+	myContainer.innerHTML += `
+	<ul class='list-container'>
+		<li class='list-items'>
+		<input type="checkbox" id="check">
+		${item.description}
+		</li>
+	</ul>
+`;
+});
+
+myContainer.innerHTML += `
+<div class='footer-list'>
+	<a href='#' class='footer-a'>Clear all completed</a>
+</div>
+`;
+
+
+const formContainer = document.getElementById('form')
+const inputElement = document.getElementById('input-element');
+const myContainer = document.getElementById('items-container');
+const clearSelectedList = document.getElementById('footer-a')
+
+
+
+const loadTasks = () => {
+	if (localStorage.getItem("listItems") == null){
+		return;
+	}
+	const toDoList = Array.from(JSON.parse(localStorage.getItem("listItems")));
+
+	toDoList.forEach((item) => {
+		const list = document.createElement("li");
+		list.classList.add('to-do-list-elements')
+		list.innerHTML = `
+			<input type="checkbox" onclick="taskCompleted(this)" class="check">
+			<input type="text" value="${item.description}" class="task ${item.completed ? 'completed' : ''}" onfocus="getCurrentTask(this)" onblur="editTaskItem(this)">
+			<i class="fa fa-trash" onclick='removeItem(this)'></i>
+		`;
+		myContainer.insertBefore(list, myContainer.children[0]);
+	});
+}
+
+
+const addToDoItem = () => {
+	let elemList = []
+	if(inputElement.value === ''){
+		return false;
+	}
+	if (document.querySelector(`input[value="${inputElement.value}"]`)){
+		return false;
+	}
+	elemList.push(inputElement.value)
+	localStorage.setItem("listItems", JSON.stringify([...JSON.parse(localStorage.getItem("listItems") || "[]"), { description: inputElement.value, completed: false, index: elemList.length}]));
+
+	const list = document.createElement("li");
+	list.classList.add('to-do-list-elements')
+	list.innerHTML = `
+
+		<input type="checkbox" onclick="taskCompleted(this)" class="check">
+		<input type="text" value="${inputElement.value}" class="task" onfocus="getCurrentTask(this)" onblur="editTaskItem(this)">
+		<i class="fa fa-trash" onclick='removeItem(this)'></i>
+
+	`;
+	myContainer.insertBefore(list, myContainer.children[0]);
+	// clear input
+	inputElement.value = "";
+}
```

Comment:
- [ ] Nothing happens when users add tasks. Please ensure the task is displayed on the UI and saved in the local storage after clicking on submit button.